src/shared/clock-util.c

```diff
@@ -137,12 +137,15 @@ int clock_reset_timewarp(void) {
         return 0;
 }
 
-#define EPOCH_FILE "/usr/lib/clock-epoch"
+#define EPOCH_FILE "/var/lib/systemd/clock-epoch"
```

Comment:
Why? This is a build-time epoch, it's supposed to be in the read-only vendor subsystem

Reply:
My bad, I was planning to update the timestamp in this file periodically and started thinking of it like changeable file. I'll revert this change.

Reply:
@Blarse timesyncd maintains a dynamic touch file already, that it touches whenever it gets an NTP sync.

this static file here should remain in /usr